Makefile

```diff
@@ -49,10 +50,11 @@ all_platforms: vendor
 .PHONY: install
 install: $(NAME)
 	sudo mv $(NAME) $(INSTALL_PATH)
+	mkdir -p $(CONFIG_FILE_DIR)
 	cp -i $(CONFIG_FILE_EXAMPLE) $(CONFIG_FILE_TARGET)
 	@echo "You need to edit $(CONFIG_FILE_TARGET) with your credentials"
 
 .PHONY: uninstall
 uninstall:
 	sudo rm $(INSTALL_PATH)/$(NAME)
-	rm $(CONFIG_FILE_TARGET)
\ No newline at end of file
+	rm $(CONFIG_FILE_TARGET)
```

Comment:
Don't you think it should also delete the directory we created?

Reply:
I should do that. I've fixed it.